README.md

```diff
@@ -1,21 +1,21 @@
 # ConstrOpt
+ConstrOpt is a library that leverages application data constraints to optimize database-backed web applications. You can find the related paper [here](https://arxiv.org/abs/2205.02954).
+
+## Run Experiment
```

Comment:
It's better to make appname a parameter

Reply:
I'm not sure what you mean here, do you mean make "ConstrOpt" shown as code format? (e.g. `ConstrOpt`)

Reply:
Can you change pipeline instruction to `python3 autrite/pipeline.py appname`?

Reply:
fixed